### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by AlaaAlsalem**
Here are the things that we Implemented to this project
>Implement  basic UI with plain HTML
>Create a collection that keeps a list of books
>Create a function to add a new book to the collection, with title and author.
>Create a function to remove a book from the collection
>Create a function to display all books saved in the collection in the top part of the page

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+let bookList = JSON.parse(localStorage.getItem('books')) || [];
+
+function addBook(title, author) {
+  bookList.push({ title, author });
+  localStorage.setItem('books', JSON.stringify(bookList));
+}
+
+function removeBook(element, title, author) {
```
Comment (rloterh) [inline script.js:3]: - [ ] The **_removeBook()_** method is not working as intended. When the same book title and book author are entered twice or more and subsequently one of them is deleted, both all entries add with the same title and the author gets removed as well. We actually want each entry to have a **unique id**, or perhaps you can improve upon this aspect by **preventing duplicates**

File: script.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+let bookList = JSON.parse(localStorage.getItem('books')) || [];
+
+function addBook(title, author) {
```
Comment (rloterh) [inline script.js]: - [ ] OPTIONAL
The book _title_ and book _author_ input are both accepting numbers as inputs. Kindly endeavor to provide validation for the same

Comment (rloterh) [inline script.js]: - [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

File: index.html [hunk 3]
```diff
@@ -4,18 +4,18 @@
...
-  </main>
+  <h1>Awesome books</h1>
+	<div class="books"></div>
+	<div>
+		<input type="text" id="title" name="title" placeholder="Title">
+		<input type="text" id="author" name="author" placeholder="Author">
+		<button type="button" class="add-book-btn">Add</button>
+	</div>
+  <script src="script.js"></script>
+
```
Comment (rloterh) [inline index.html]: - [ ] OPTIONAL
Your code is not properly indented. Kindly ensure that your code has consistent **spacing** for indentation. The recommended spacing is **2 spaces**. You can install and configure a code formatter plugins such as [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify) in your code editor for formatting your code to ensure consistent and well-indented code.

File: script.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+  div.appendChild(removeBtn);
+
+  booksCollection.appendChild(div);
+});
+
+const addBtn = document.querySelector('.add-book-btn');
+
+addBtn.addEventListener('click', () => {
+  // add book
```
Comment (rloterh) [inline script.js:45]: - [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Aladdin & Sly, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**